recipe/meta.yaml

```diff
@@ -21,23 +21,25 @@ outputs:
         - black = black:patched_main
     requirements:
       host:
-        - python >=3.6
```

Comment:
Why was the version constraint removed? I thought black is Python 3 only?

Reply:
True, but that is now also the oldest version of python still supported by conda-forge?

Reply:
Yes but people that still use Python 2.7 in some conda environments and removing this constraint will lead to successful solves there as this is a `noarch: python` package. You can only omit the constraint on conda-forge on non-noarch packages.